apps/docs/src/app/core/component-docs/dialog/examples/stacked-dialogs/second-dialog-example.component.ts

```diff
@@ -6,7 +6,7 @@ import { DialogRef } from '@fundamental-ngx/core/dialog';
         <fd-dialog>
             <fd-dialog-header>
                 <h1 id="fd-dialog-header-9" fd-title>Second Dialog</h1>
-                <button fd-dialog-close-button (click)="dialogRef.dismiss()"></button>
+                <button fd-dialog-close-button (click)="dialogRef.dismiss()" i18n-title="@@dialogPlayGround.closebutton.title"></button>
```

Comment:
`i18n-title` not needed here as explained above. `title` is needed.

Reply:
fixed